pkg/filter/filter.go

```diff
@@ -103,6 +103,11 @@ func filterByPattern(entity string, include, exclude []regex.Regex) error {
 // wakatime project file can be detected in the filepath directory tree.
 // Returns an error to signal to the caller to skip the heartbeat.
 func filterFileEntity(heartbeat heartbeat.Heartbeat, config Config) error {
+	// skip filters for remote files downloaded to a local tmp file
+	if heartbeat.IsRemote() {
```

Comment:
It could be moved upwards to replace lines 59-66 in this file.
```suggestion
func Filter(h heartbeat.Heartbeat, config Config) error {
    // omitted for brevity
    
    // skip filters for remote files downloaded to a local tmp file
    if heartbeat.IsRemote() {
	return nil
    }
    
    err := filterFileEntity(h, config)
    // omitted for brevity
}
```